x11docker

```diff
@@ -4705,14 +4711,14 @@ DefaultEnvironment=DISPLAY=$Newdisplay XAUTHORITY=$Cshare/Xclientcookie $(IFS=$'
       echo "    note 'Want to create and mount a cgroup for $Wantcgroup.
   As x11docker currently does not run as root, this will probably fail.
   Please either run x11docker as root, or run with option --pw=su or --pw=sudo.
-  
+
```

Comment:
need spaces here instead of empty line. Messages with empty lines are handled differently and add a sleep due to a workaround for MSYS2 (#55)